index.html

```diff
@@ -0,0 +1,269 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
+    <link rel="stylesheet" href="style.css">
+    <title>My Personal Portifolio</title>
+</head>
+<body>
+   <header class="header-container">
+
+        <nav class="navbar">
+
+            <a href="#"><img class="my-logo" src="images/Cizzla_logo.png" alt=""></a>
+
+
+            <ul class="nav-links">
+                <li><a class="links" href="#">Portifolio</a></li>
+                <li><a class="links" href="#">About</a></li>
+                <li><a class="links" href="#">Contact</a></li>
+                <li><a class="links" href="#"><img src="images/Icon - Mail.svg" alt=""></a></li>
+            </ul>
+
+                <i class="fas fa-bars"></i>
+        </nav>
+
+        <main class="main-container">
+
+            <ul class="social-icons">
+                <li><a href=""><img class="icons" src="images/github@2x.png" alt=""></a></li>
+                <li><a href=""><img class="icons" src="images/Linkedin icon@2x.png" alt=""></a></li>
+                <li><a href=""><img class="icons" src="images/angel list@2x.png" alt=""></a></li>
+                <li><a href=""><img class="icons" src="images/twitter@2x.png" alt=""></a></li>
+                <li><a href=""><img class="icons" src="images/medium@2x.png" alt=""></a></li>
+
+            </ul>
+
+            <div class="main-info">
+                <h1 class="info-header">Hey There.<br>I'm Ian.</h1>
+                <p class="info-title">I'm a Software Developer</p>
+                <p class="info-details">I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.</p>
+
+            </div>
+
+        </main>
+   </header>
+
+   <!-- Recent Work Section-->
+   <section class="work-container">
+
+        <div class="project-title">
+            <div class="header-title">
+                <h1>My Recent Works</h1>
+            </div>
+            <div class="line-container">
+                <hr>
+             </div>
+        </div>
+
+
+        <div class="main-project">
+
+            <div class="main-img">
+                <img src="images/Img Placeholder.png" alt="">
+            </div>
+
+            <div class="main-content">
+                <h2>Multi-Post Stories</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul>
+                    <li>CSS</li>
+                    <li>HTML</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
```

Comment:
- Kindly, would arrange the right margin of your recent framework list to stick to the Figma design and have a better UI 👍  
![Uploading Screenshot 2023-06-05 123338.png…]()